src/main/java/net/openhft/chronicle/wire/LongConverter.java

```diff
@@ -63,8 +96,17 @@ default void lengthCheck(CharSequence text) {
             throw new IllegalArgumentException(format("text={0} exceeds the maximum allowable length of {1}", text, maxParseLength()));
     }
 
-    static int maxParseLength(int based) {
-        return (int) (64 / log(based) * log(2));
+    default boolean allSafeChars() {
+        return true;
```

Comment:
"safe" is very contextual. I assume this means safe for YAML strings from your explanation, perhaps this should be named more specifically. (e.g. XML has a separate set of characters that need to be escaped, JSON different ones and windows/linux filenames different again).

Either that or it could be generic and we could provide an enum like `allSafeChars(ForPurpose.WINDOWS_FILENAME)` ?

Reply:
I have added a parameter so the type of the WireOut can be used to determine whether they are safe or not.